expat/cmake/autotools/expat-noconfig__windows.cmake.in

```diff
@@ -9,11 +9,11 @@ set(CMAKE_IMPORT_FILE_VERSION 1)
 set_property(TARGET expat::expat APPEND PROPERTY IMPORTED_CONFIGURATIONS NOCONFIG)
 set_target_properties(expat::expat PROPERTIES
   IMPORTED_IMPLIB_NOCONFIG "${_IMPORT_PREFIX}/@LIBDIR_BASENAME@/libexpat.dll.a"
-  IMPORTED_LOCATION_NOCONFIG "${_IMPORT_PREFIX}/bin/libexpat-@SO_MAJOR@.dll"
+  IMPORTED_LOCATION_NOCONFIG "${_IMPORT_PREFIX}/bin/@CMAKE_SHARED_LIBRARY_PREFIX@expat-@SO_MAJOR@.dll"
   )
 
 list(APPEND _IMPORT_CHECK_TARGETS expat::expat )
-list(APPEND _IMPORT_CHECK_FILES_FOR_expat::expat "${_IMPORT_PREFIX}/@LIBDIR_BASENAME@/libexpat.dll.a" "${_IMPORT_PREFIX}/bin/libexpat-@SO_MAJOR@.dll" )
+list(APPEND _IMPORT_CHECK_FILES_FOR_expat::expat "${_IMPORT_PREFIX}/@LIBDIR_BASENAME@/libexpat.dll.a" "${_IMPORT_PREFIX}/bin/@CMAKE_SHARED_LIBRARY_PREFIX@expat-@SO_MAJOR@.dll" )
```

Comment:
There is a verbatim `libexpat` in this line that probably need to be `@CMAKE_SHARED_LIBRARY_PREFIX@expat` now? One more place like that in line 11, too.

Reply:
@carlo-bramini maybe that^^ was mistaken. I see there is `CMAKE_STATIC_LIBRARY_PREFIX` too and `.dll.a` is counted as static, I guess? Does Cygwin mix `cygexpat-1.dll` with `libexpat.dll.a` or `cygexpat.dll.a`? :thinking:

Reply:
@carlo-bramini I think I found the answer to that at https://cygwin.com/cygwin-ug-net/dll.html :
> The name of your library is ${module}, prefixed with cyg for the DLL and lib for the import library.
So I guess you're doing the right thing already. I'll mark this as resolved.